### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by guelz** [BJReplay/ha-solcast-solar#111] [General]
Thank you for continuing this integration! I couldn't do anything (useful) without it!
Anyway, found a spelling error!
![ksnip_20240805-125400](https://github.com/user-attachments/assets/3f627150-119f-4cbe-a8a0-fcba25e70709)
This should definitly read: "Depends on the cloud**s**" !-)

**Comment by autoSteve** [comment]
Many thanks. I do believe we're taking this integration next-level reliable, plus I'm loving the new features. Jump on board with `Show beta versions` in HACS re-download for some new things, and report back in the beta discussion!

Whilst I agree that "the cloud" is just someone else's computer running somewhere on the Internet, and there will be many of those computers doing many different things, it has rightly or wrongly become customary to refer to this "someone else's stuff" as "the cloud".

It boils my piss dry gramatically, but it is what it is. 🤣

Being real nit-picky, it is just one cloud service being relied on in this case. Solcast.

**Comment by autoSteve** [comment]
OMG. I just got your joke. LOLS. 😜🤣🤣

**Reply by autoSteve** [reply]
Sadly this is unchangeable, or it "Depends on the cloud(s)" would deffo have been in v4.0.42. Unfortunately we've only got to work with `"iot_class": "cloud_polling"`. Damn.
